sort.java

```diff
@@ -21,7 +21,7 @@ void display(int arr[]) {
 
     public static void main(String[] args) {
         Sort ob = new Sort();
```

Comment:
Please have descriptive object names. I'd suggest renaming this to `sortObject` or just `sort`.